### DISCUSSION TITLE
Is it possible to support Webhook?

### DISCUSSION THREAD

**Discussion by db-one** [crazy-max/docker-fail2ban#142] [Q&A]
Is it possible to support Webhook?
To make it easier for https://api.telegram.org to receive notification messages
Thank you very much

**Comment by schklom** [comment]
You can customize actions with `/data/actions.d/sendmail-common.local`.
The default is at `/etc/fail2ban/sendmail-common.conf`.

All you need is to call your webhook in python.

Alternatively, set up [Apprise-API](https://github.com/caronc/apprise-api) and [Mailrise](https://github.com/YoRyan/mailrise) to use any notification system you want. You send notification emails to Mailrise triggering calls to Apprise which can call most notification systems.
